create_icr_file/create_icr_file.py

```diff
@@ -0,0 +1,66 @@
+import argparse
+import os
+import random
+
+from core_data_modules.traced_data.io import TracedDataJsonIO, TracedDataCSVIO
+from core_data_modules.util import IOUtils
+
+# Number of messages to export in the ICR file
+ICR_MESSAGES_COUNT = 200
+# Number of characters below which messages are considered noise
+NUMBER_OF_CHARS_NOISE = 15
```

Comment:
Where does this number come from?

Reply:
15? It's what we used in the CMS. It's an arbitrary number.